WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -139,6 +141,7 @@ public static void viewAccountSettings(Context context) {
 
     public static void viewHelpAndSupport(Context context) {
         Intent intent = new Intent(context, HelpActivity.class);
+        intent.putExtra(HelpshiftHelper.ORIGIN_KEY, Tag.ORIGIN_ME_SCREEN_HELP);
```

Comment:
This assumes `viewHelpAndSupport()` will only be called from the "Me" tab, which is true now but may not be in the future. Perhaps the origin tag should be passed to this routine instead?